src/fix_wall_region.h

```diff
@@ -38,12 +38,12 @@ class FixWallRegion : public Fix {
   double compute_scalar() override;
   double compute_vector(int) override;
 
- private:
+ protected:
   int style;
   double epsilon, sigma, cutoff;
   double alpha;
   int eflag;
-  double ewall[4], ewall_all[4];
+  double *ewall, ewall_all[4]; // need ewall double*, not double[] for kokkos dual view
```

Comment:
This isn't really necessary, if you copy the elements manually. @akohlmey do you care?

Reply:
> if you copy the elements manually. @akohlmey do you care?

If the change can be kept local within the KOKKOS version, that is preferred.
For consistency, either both arrays should be allocated dynamically or  with fixed size in the class.
For 4 elements a manual copy is still acceptable IMO. If I remember correctly, it is done even for the virial in places with 6 elements.

Reply:
done